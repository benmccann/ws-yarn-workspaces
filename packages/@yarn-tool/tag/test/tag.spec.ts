describe(`throw`, () =>
{

	test(`empty`, () =>
	{

		let actual;
		let expected;

		expect(actual).toStrictEqual(expected);
		//expect(actual).toBeInstanceOf(Date);
		expect(actual).toMatchSnapshot();

	});

})
