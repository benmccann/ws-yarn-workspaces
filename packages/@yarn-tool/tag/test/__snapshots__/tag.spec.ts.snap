// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`throw empty 1`] = `undefined`;
